frontend/common/stores/account-store.js

```diff
@@ -328,12 +329,20 @@ const controller = {
     } else if (!user) {
       store.ephemeral_token = null
       AsyncStorage.clear()
-      API.setCookie('t', '')
-      data.setToken(null)
-      API.reset().finally(() => {
-        store.model = user
-        store.organisation = null
-        store.trigger('logout')
+      if (!data.token) {
+        return
+      }
+      ;(Project.cookieAuthEnabled
```

Comment:
... is the semi-colon here some horrible JS syntax that I'm not aware of, or a typo? @kyle-ssg